integreat_compass/cms/forms/offers/offer_version_form.py

```diff
@@ -82,7 +77,7 @@ def clean(self):
         """
         cleaned_data = super().clean()
         if cleaned_data["title_image"] is False:
-            cleaned_data["title_image"] = settings.DEFAULT_TITLE_IMAGE
+            cleaned_data["title_image"] = ""
```

Comment:
```suggestion
            cleaned_data["title_image"] = None
```
Wouldn't this be `None` instead of an empty string?

Reply:
`None` works (in the sense that no error is thrown), nut removing images stops working. I think `None` fields are just not passed to the model (?).

Reply:
Ah ok I get it, the way how it is supposed to work is setting `checked = true` on the hidden `offer_version-title_image-clear_id` checkbox in JS when clearing the input. Then, we don't need any custom validation at all in the form class,

Reply:
Ah, I just noticed this is already the case? So then we can probably remove the custom clean method?

Reply:
Oh - indeed. Must have become superfluous at some point I guess :D